### DISCUSSION TITLE
wix 3.11.2 System.Management Issue

### DISCUSSION THREAD

**Discussion by NimaEagle** [wixtoolset/issues#6689] [Questions]
Hi
I'm developing an application is in .net-5.0 and is sdk-style on visual studio 2019 inside windows 10,
i've included all of my dependency projects to the product file of wix installer and built the installer and installed the app successfully,
but when i run the app i get the "System.Management currently is only supported for Windows desktop applications" error.
i checked the the structure there is System.Management.dll in installer.
i think this happens because i didn't include runtimes (runtimes\win\lib\netstandard2.0\System.ServiceProcess.ServiceController.dll and \runtimes\win\lib\netcoreapp2.0\System.Management.dll) because when i run the app directly in bin\release folder of project that have runtimes i don't get the indicated error.
if my guess is right, i dont know how to include runtimes to wix installer.

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh]. You might also consider asking your question on the [wix-users mailing list][ml] or posting on [Stack Overflow][so].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
[ml]: http://wixtoolset.org/documentation/mailinglist/
[so]: https://stackoverflow.com/questions/tagged/wix

**Comment by TrulsHaraldsson** [comment]
Maybe you have to install the runtime, as you say, on the system you are installing the app? 
https://dotnet.microsoft.com/en-us/download/dotnet/5.0

And you can make use of a Bundle to install that runtime before the app. See https://wixtoolset.org/documentation/manual/v3/bundle/
